packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -121,13 +125,16 @@ import {
   hasSuspenseContext,
   setDefaultShallowSuspenseContext,
   addSubtreeSuspenseContext,
+  setShallowSuspenseContext,
 } from './ReactFiberSuspenseContext';
+import {isShowingAnyFallbacks} from './ReactFiberSuspenseComponent';
 import {
   pushProvider,
   propagateContextChange,
   readContext,
   prepareToReadContext,
   calculateChangedBits,
+  scheduleWorkOnParentPath,
```

Comment:
The other method does more things and doesn't bailout until it hits the root since it needs to return the root and schedule work on the root and possibly with scheduler. This path doesn't need that. So I think I'll leave this in and hope that the VM can inline it instead.